### PR TITLE
Upgrade harden package

### PR DESCRIPTION
**Pull request by michaelfig**
The new one both works and has Typescript types.

### CODE REVIEW

**Review by katelynsills — APPROVED** [review]
Looks good!

File: yarn.lock [hunk 1]
```diff
@@ -7,13 +7,20 @@
   resolved "https://registry.yarnpkg.com/@agoric/babel-parser/-/babel-parser-7.6.4.tgz#8dd5d36554cc758c29042713b5aa57dc58ee5273"
   integrity sha512-3FQC3eP2hekhz1zn+2LcSL7tAG2dGgSqbmCeRfIFqhVS8bbE1hR7EvrC6jYlvqdU6yzlly43VykyRy9MHBvUAw==
 
-"@agoric/harden@0.0.4", "@agoric/harden@^0.0.4":
+"@agoric/harden@^0.0.4":
```
Comment (Chris-Hibbert) [inline yarn.lock:10]: Do we need to keep 0.0.4 around? it looks like all of the `packages/*/package.json` files have been updated to remove the old reference.

[I admit I don't know what yarn.lock is supposed to include.]

**Review by Chris-Hibbert — APPROVED** [review]
Looks good to me, with one question.

File: yarn.lock (hunk 1 above)
Comment (michaelfig) [reply yarn.lock:10]: I don't know why this is.  Perhaps a published dependency needs it.  Merging anyway.

**Pull request merged**